DashKitUI/Views/MockPodSettingsView.swift

```diff
@@ -70,12 +73,36 @@ struct MockPodSettingsView: View {
                             self.selectedAlert = nil
                         },
                         .default(Text("Issue in 15s")) { DispatchQueue.main.asyncAfter(deadline: .now() + 15) {
-                            print("Delayed issue of \(selectedAlert!.podAlerts) at \(Date())")
                             self.model.issueAlert(selectedAlert!.podAlerts)
                         } },
                     ]
                 )
             }
+            Section(header: Text("Alarms").font(.headline).foregroundColor(Color.primary)) {
+                ForEach(SimulatedPodAlarm.allCases, id: \.self) { item in
+                    Button(action: {
+                        self.selectedAlarm = item
+                        self.showAlarmActions = true
+                    }) {
+                        Text("\(item.rawValue)")
+                    }
+                }
+            }
+            .actionSheet(isPresented: $showAlarmActions) {
+                ActionSheet(
+                    title: Text("\(selectedAlarm!.rawValue)"),
+                    buttons: [
+                        .cancel(),
+                        .default(Text("Issue Immediately")) {
+                            self.model.triggerAlarm(selectedAlarm!)
+                            self.selectedAlert = nil
+                        },
+                        .default(Text("Issue in 15s")) { DispatchQueue.main.asyncAfter(deadline: .now() + 15) {
+                            self.model.triggerAlarm(selectedAlarm!)
```

Comment:
Should this also deselect the alert after it is triggered?

Reply:
Not needed; the selected alarm is part of the alarm action sheet; it just needs to be set correctly anytime that is shown.